projects/addon-tablebars/components/table-bars-host/test/table-bars-host.component.spec.ts

```diff
@@ -35,23 +34,24 @@ describe('TableBarsHost', () => {
         },
     };
 
-    configureTestSuite(() => {
-        TestBed.configureTestingModule({
-            imports: [TuiTableBarsHostModule, NoopAnimationsModule],
-            declarations: [TestComponent],
-            providers: [TuiTableBarsService],
-        });
-    });
-
-    beforeEach(() => {
-        fixture = TestBed.createComponent(TestComponent);
-        pageObject = new PageObject(fixture);
-        testComponent = fixture.componentInstance;
-        fixture.detectChanges();
-        component = testComponent.component;
-        service = TestBed.inject(TuiTableBarsService);
-        service.bar$['_events'] = [];
-    });
+    configureTestSuite(
+        () => {
+            TestBed.configureTestingModule({
+                imports: [TuiTableBarsHostModule, NoopAnimationsModule],
+                declarations: [TestComponent],
+                providers: [TuiTableBarsService],
+            });
+        },
+        () => {
+            fixture = TestBed.createComponent(TestComponent);
+            pageObject = new PageObject(fixture);
+            testComponent = fixture.componentInstance;
+            fixture.detectChanges();
+            component = testComponent.component;
+            service = TestBed.inject(TuiTableBarsService);
+            service.bar$['_events'] = [];
+        },
```

Comment:
I don't think we need to put it in a callback. if it stays also in this place, nothing bad will happen!

```ts
 beforeEach(() => {
        fixture = TestBed.createComponent(TestComponent);
        pageObject = new PageObject(fixture);
        testComponent = fixture.componentInstance;
        fixture.detectChanges();
        component = testComponent.component;
        service = TestBed.inject(TuiTableBarsService);
        service.bar$['_events'] = [];
    });
```